src/components/ActionsPanel/constants.ts

```diff
@@ -0,0 +1,209 @@
+      to: EXT_URLS.LEDGER_REFERRAL.url,
+      external: true
+    },
+    category: ACTION_CATEGORIES.SECURITY
+  },
+  {
+    name: ACTION_NAME.MYC_MEMBERSHIP,
+    heading: translateRaw('MYC_MEMBERSHIP_ACTION_HEADING'),
+    icon: 'membership',
+    body: [translate('MYC_MEMBERSHIP_ACTION_BODY')],
+    filter: (state: StoreContextState) => !state.isMyCryptoMember,
+    priority: 0,
+    button: {
+      content: translateRaw('MYC_MEMBERSHIP_ACTION_BUTTON'),
+      to: ROUTE_PATHS.MYC_MEMBERSHIP.path,
+      external: false
+    },
+    category: ACTION_CATEGORIES.SELF_LOVE
+  },
+  {
+    name: ACTION_NAME.ADD_ACCOUNT,
+    heading: translateRaw('ADD_ACCOUNT_ACTION_HEADING'),
+    icon: 'experience',
+    body: [translate('ADD_ACCOUNT_ACTION_BODY')],
+    filter: (state: StoreContextState) => state.accounts.length < 3,
+    priority: 0,
+    button: {
+      content: translateRaw('ADD_ACCOUNT_ACTION_BUTTON'),
+      to: ROUTE_PATHS.ADD_ACCOUNT.path,
+      external: false
+    },
+    category: ACTION_CATEGORIES.MYC_EXPERIENCE
+  },
+  {
+    name: ACTION_NAME.BACKUP,
+    heading: translateRaw('BACKUP_ACTION_HEADING'),
+    icon: 'experience',
+    body: [translate('BACKUP_ACTION_BODY')],
+    filter: (state: StoreContextState) => state.accounts.length >= 3,
+    priority: 0,
+    button: {
+      content: translateRaw('BACKUP_ACTION_BUTTON'),
+      to: ROUTE_PATHS.SETTINGS_EXPORT.path,
+      external: false
+    },
+    category: ACTION_CATEGORIES.MYC_EXPERIENCE
+  },
+  {
+    name: ACTION_NAME.FEEDBACK,
+    heading: translateRaw('FEEDBACK_ACTION_HEADING'),
+    icon: 'feedback',
+    body: [translate('FEEDBACK_ACTION_BODY_1'), translate('FEEDBACK_ACTION_BODY_2')],
+    priority: 0,
+    button: {
+      content: translateRaw('FEEDBACK_ACTION_BUTTON'),
+      to: formatSupportEmail(translateRaw('FEEDBACK_ACTION_MAIL_SUBJECT')),
+      external: true
+    },
+    category: ACTION_CATEGORIES.OTHER
+  },
+  {
+    name: ACTION_NAME.NEWSLETTER,
+    heading: translateRaw('NEWSLETTER_ACTION_HEADING'),
+    icon: 'newsletter',
+    body: [translate('NEWSLETTER_ACTION_BODY')],
+    priority: 0,
+    button: {
+      content: translateRaw('NEWSLETTER_ACTION_BUTTON'),
+      to: SUBSCRIBE_NEWSLETTER_LINK,
+      external: true
+    },
+    category: ACTION_CATEGORIES.OTHER
+  },
+  {
+    name: ACTION_NAME.TELEGRAM,
+    heading: translateRaw('TELEGRAM_ACTION_HEADING'),
+    icon: 'telegram-icon',
+    body: [translate('TELEGRAM_ACTION_BODY')],
+    priority: 0,
+    button: {
+      content: translateRaw('TELEGRAM_ACTION_BUTTON'),
+      to: socialMediaLinks.find((i) => i.text === 'telegram')!.link,
+      external: true
+    },
+    category: ACTION_CATEGORIES.OTHER
+  },
+  {
+    name: ACTION_NAME.SWAP,
+    heading: translateRaw('SWAP_ACTION_HEADING'),
+    icon: 'swap',
+    body: [translate('SWAP_ACTION_BODY')],
+    filter: (state: StoreContextState) =>
+      state.assets().some((a) => a.uuid === ETHUUID) &&
```

Comment:
Wouldn't it be enough to just check if they have any main net assets?

Reply:
The filter for this card is if the user has any eth AND token balance so we need to check for both

Reply:
Ah, yeah that makes sense!